### ISSUE TITLE
CORS error while browser devtools are closed

### ISSUE THREAD

**Issue by oleksii-shaposhnikov** [axios/axios#3557]
#### Describe the issue
General description:
I am trying to fetch the assets (images (.png), sounds(.mp3) and videos (.mp4)) using Axios as a blob. That is my code:

```
import axios from 'axios';

axios({
  method: 'GET',
  responseType: 'blob',
  url: '' // asset cdn url,
})
// ...
```
That works when I open the browser devtools (all requests have status 200).
And that fails (with common CORS error - `Access to XMLHttpRequest at 'cdn url' from origin 'app url' has been blocked by CORS policy: `No 'Access-Control-Allow-Origin' header is present on the requested resource.`) when the browser dev tools are closed
The CORS are correctly set, so this is not a reason for the issue.

The issue seems to be very strange since opened dev tools don't change the requests. Do you have any ideas?

#### Environment
- Axios Version ^0.19.0
- Browser Chrome
- Browser Version 87.0.4280.141
- Node.js Version v14.15.1
- OS: MacOS Big Sur 11.1

**Comment by thanoseleftherakos** [comment]
exact same problem here

**Comment by jasonsaayman** [comment]
Try the same in firefox, chrome is overly zealous with its cors errors, also please send a screenshot of the dev tools output

**Comment by jasonsaayman** [comment]
Hi 👋 

Please could you retry with the latest version and open a new issue should this error still be relevant?

Thanks

**Comment by madvocate** [comment]
@oleksii-shaposhnikov did you ever figure this one out? We just hit this issue but we are not even using axios (we use fetch) and this is exactly what is happening - when devtools are closed, we're getting CORS issues (from object storage signed URL) but all works when devtools are open and Disable Cache is checked on the Network Tab. So weird. Almost feels like a browser issue of some sort but the common denominator is that we also use the .blob() method on the response from fetch!
